crates/anvil/src/cmd.rs

```diff
@@ -297,7 +297,13 @@ impl NodeArgs {
             let mut rng = rand_08::thread_rng();
             let mnemonic = match Mnemonic::<English>::new_with_count(&mut rng, count) {
                 Ok(mnemonic) => mnemonic.to_phrase(),
-                Err(_) => DEFAULT_MNEMONIC.to_string(),
+                Err(err) => {
+                    warn!(target: "node", ?count, %err, "invalid mnemonic word count, falling back to 12-word random mnemonic");
```

Comment:
this error message is not correct, it might also be that the word is not in the dictionary